lib/rspec/core/formatters.rb

```diff
@@ -168,9 +168,9 @@ def add(formatter_to_use, *paths)
 
     def find_formatter(formatter_to_use)
       built_in_formatter(formatter_to_use) ||
-      custom_formatter(formatter_to_use)   ||
-      (raise ArgumentError, "Formatter '#{formatter_to_use}' unknown - " \
-                            "maybe you meant 'documentation' or 'progress'?.")
```

Comment:
Did this whitespace cause the rubocop to fail? Its a little easier to read when these are aligned with the quotes.